<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конфигуратор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="PortLayout">
      <item>
       <widget class="QComboBox" name="CmbPorts">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblPortStatus">
        <property name="styleSheet">
         <string notr="true">color: green;</string>
        </property>
        <property name="text">
         <string>Обнаружен.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnPortChange">
        <property name="text">
         <string>Сменить порт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="LblWifiSsid">
        <property name="text">
         <string>Название сети</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxtWifiSsid">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>TP-Link_2A24</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblWifiPassword">
        <property name="text">
         <string>Пароль сети</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxtWifiPassword">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>393-63-461</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblMac">
        <property name="text">
         <string>MAC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxtMac">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>94:3C:C6:38:72:40</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblWifiStatus">
        <property name="styleSheet">
         <string notr="true">color: green;</string>
        </property>
        <property name="text">
         <string>Подключено.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnWifiConnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnWifiChange">
        <property name="text">
         <string>Сменить сеть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="LblUserLogin">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxtUserLogin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>admin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblUserPassword">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxtUserPassword">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>admin</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblUserStatus">
        <property name="styleSheet">
         <string notr="true">color: green;</string>
        </property>
        <property name="text">
         <string>Авторизован.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnLogin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnChangeUser">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Сменить пользователя</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="LblMcKey">
        <property name="text">
         <string>Ключ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxtMcKey">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>GOIH8TT93UGRV7MV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnGetMcKey">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Перерегистрация</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
